Alicerce/Sources/Logging/Log+NodeLogDestination.swift

```diff
@@ -12,15 +12,22 @@ public extension Log {
 
     public class NodeLogDestination : LogDestination {
 
+        public struct Errors: Error {
```

Comment:
The idiomatic way to declare error types in swift is simply using `enum`. If you want to namespace types inside each other the best way is to create a caseless `enum`.

However, in this case I think we only need a single enum. We can and should also take advantage of enum's associated values, so in your case a possible error enum could be: 

```swift

enum Error: Swift.Error { // simply call it Error because it is namespaced inside `NodeLogDestination`
    case httpError(statusCode: Int)
    case network(Error)
}

```